.github/workflows/docker.yml

```diff
@@ -105,7 +120,8 @@ jobs:
         with:
           context: .
           file: _dev/docker/mono/Dockerfile
-          tags: ${{ env.GAR_LOCATION }}-docker.pkg.dev/${{ env.GCP_PROJECT_ID }}/${{ env.GAR_REPOSITORY}}/${{ env.IMAGE}}:${{ env.GIT_TAG }}
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
```

Comment:
Where is labels coming from?

Reply:
It comes from the docker/metadata-action https://github.com/docker/metadata-action?tab=readme-ov-file#overwrite-labels-and-annotations